src/style.css

```
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  font-family: Roboto;
```

Не забывай добавлять альтернативные стили, если Гугл фонтс работать не будеь
```suggestion
  font-family: Roboto, Arial, Helvetica, sans-serif;
```